infrastructure/terraform/modules/storageaccount/output.tf

```diff
@@ -15,3 +15,9 @@ output "storage_account_primary_blob_endpoint" {
   description = "Specifies the primary blob endpoint of the storage account"
   sensitive   = false
 }
+
+output "storage_account_primary_connection_string" {
+  value       = azurerm_storage_account.storage.primary_connection_string
+  description = "Specifies the primary blob endpoint of the storage account"
+  sensitive   = true
+}
```

Comment:
I worked on a solution to generate SAS using HTTP. Also, this does not use account level SAS keys. I consider this more secure. What are your thoughts? Let's sync on that.